docs/src/domains/cdn/_index.md

```diff
@@ -5,179 +5,131 @@ weight: 3
-you need to provide your own from an issuer of your choice that can then be installed for you.
-
-If you need to provide your own TLS certificate, place the certificate, the unencrypted private key,
-and the necessary certificate chain supplied by your TLS provider in your app's `private` directory (not web accessible),
-and then open a ticket to let our team know to install it.
-
-Dedicated plans support a single TLS certificate on the origin.
-Support for multiple certificates is offered only through a CDN such as CloudFront or Fastly.
-Self-signed certificates can optionally be used on the origin for development purposes or for enabling TLS between the CDN and origin.
-
-All TLS certificates used with CloudFront MUST be 2048 bit certificates.
-Larger sizes don't work.
-
-## Host Header forwarding
-
-The `Host` HTTP header tells the server what domain name is being requested,
-which may vary when multiple domains are served from the same server or through the same proxy, as is the case with a CDN.
-But the CDN can't use the production domain name to reach Platform.sh, as that domain already routes to the CDN.
-So it uses the origin name provided by Platform.sh.
-
-To ensure your TLS certificates are valid for both requests from clients to the CDN and from the CDN to the server on Platform.sh,
-you need to take two additional steps:
-
-1. Configure your CDN to set the `X-Forwarded-Host` HTTP header to your domain, for example: `example.com`.
-   That allows the request from the CDN to Platform.sh to still carry the original requested domain.
-   The specific way to do so varies by the CDN.
-2. Ensure your app can read from the `X-Forwarded-Host` header should it need the Host information.
-   Many popular apps already do so,
-   but if you have a custom app make sure that it checks for that header
-   and uses it instead of `Host` as appropriate.
-
-## Web Application Firewall & Anti-DDoS
-
-All Platform.sh-hosted sites, either Grid or Dedicated, live on infrastructure provided by major cloud vendors.
-These vendors include their own Level 3 DDoS protection that is sufficient for the majority of cases.
-
-Customers are welcome to put their own WAF in front of a Dedicated cluster or add other security measures not included in the offering.
-
-## The router cache
-
-When using a CDN the Platform.sh router's HTTP cache becomes redundant.
-In most cases it's best to disable it outright.
-Modify your route like so to disable the cache:
-
+ 
+Using a CDN speeds up the delivery of your site's content to its users.
+The CDN deploys edge servers at many locations around the world.
+These edge servers behave like local caches to nearby users.
+Bringing content closer to users helps enhance your site's perceived performance
+and so can improve user engagement and retention.
+
+Fastly is the recommended CDN for Platform.sh projects.
+By default, Dedicated projects include a [Fastly](./fastly.md) CDN managed by Platform.sh.
```

Comment:
:lady_beetle: This refers to the managed version but links to the other version. Should be adjusted for the new page.